package.json

```
@@ -30,6 +30,7 @@
     "types": "tsc --declaration --emitDeclarationOnly && ./fixup.sh"
   },
   "dependencies": {
+    "@hide-in-shadows/html": "^0.0.5",
```

Unless we inline it.

```suggestion
    "@hide-in-shadows/html": "0.0.5",
```